wp-modules/app/package.json

```diff
@@ -9,10 +9,13 @@
   "author": "",
   "license": "GPL-2.0-or-later",
   "devDependencies": {
+    "@types/jest": "^29.4.0",
+    "@types/react-test-renderer": "^17.0.2",
```

Comment:
Good idea

Reply:
Thanks! I think we should install `@types/jest` for `editor` as well and get rid of those errors.

I would like to add these kinds of tests for the sidebar components — I can add these types in that PR.

Reply:
Oh good idea! It'd be good to add tests like these for the sidebar components.